### ISSUE TITLE
Updating the docs

### ISSUE THREAD

**Issue by rubik** [jwiegley/pipes-files#4]
The docs mention find-conduit everywhere, but it should be pipes-files, shouldn't it?

**Comment by jwiegley** [comment]
Yes, that is an omission. The docs need updating. Thanks for creating this issue to remind me!
